packages/core/factories/src/useUserOrdersFactory.ts

```diff
@@ -0,0 +1,33 @@
+import { ref, Ref, computed } from '@vue/composition-api';
+import { UseUserOrders } from '@vue-storefront/interfaces';
+
+export type OrdersSearchResult<ORDER> = {
+  data: ORDER[];
+  total: number;
+};
```

Comment:
same as with #265 - common interface for this
```suggestion
export type SearchResults<T> = {
  data: T[];
  total: number;
};
```

Reply:
Good point, I also moved it to interfaces.